src/Compilers/VisualBasic/Test/Semantic/Binding/BindingCollectionInitializerTests.vb

```diff
@@ -392,58 +397,62 @@ IBlockStatement (6 statements, 4 locals) (OperationKind.BlockStatement, IsInvali
-          Arguments(0)
-          Initializer: 
-            IObjectOrCollectionInitializerExpression (OperationKind.ObjectOrCollectionInitializerExpression, Type: C2) (Syntax: 'From {"Hello World!"}')
-              Initializers(1):
-                  ICollectionElementInitializerExpression (AddMethod: Sub C2.Add(p As System.String)) (IsDynamic: False) (OperationKind.CollectionElementInitializerExpression, Type: System.Void, IsImplicit) (Syntax: '"Hello World!"')
-                    Arguments(1):
-                        ILiteralExpression (OperationKind.LiteralExpression, Type: System.String, Constant: "Hello World!") (Syntax: '"Hello World!"')
+        ILocalInitializer (OperationKind.LocalInitializer) (Syntax: 'New C2() Fr ... lo World!"}')
+          IObjectCreationExpression (Constructor: Sub C2..ctor()) (OperationKind.ObjectCreationExpression, Type: C2) (Syntax: 'New C2() Fr ... lo World!"}')
+            Arguments(0)
+            Initializer: 
+              IObjectOrCollectionInitializerExpression (OperationKind.ObjectOrCollectionInitializerExpression, Type: C2) (Syntax: 'From {"Hello World!"}')
+                Initializers(1):
+                    ICollectionElementInitializerExpression (AddMethod: Sub C2.Add(p As System.String)) (IsDynamic: False) (OperationKind.CollectionElementInitializerExpression, Type: System.Void, IsImplicit) (Syntax: '"Hello World!"')
+                      Arguments(1):
+                          ILiteralExpression (OperationKind.LiteralExpression, Type: System.String, Constant: "Hello World!") (Syntax: '"Hello World!"')
   IVariableDeclarationStatement (1 declarations) (OperationKind.VariableDeclarationStatement, IsInvalid) (Syntax: 'Dim c As C2 ... lo World!"}')
     IVariableDeclaration (1 variables) (OperationKind.VariableDeclaration) (Syntax: 'c')
       Variables: Local_1: c As C2
       Initializer: 
-        IObjectCreationExpression (Constructor: Sub C2..ctor()) (OperationKind.ObjectCreationExpression, Type: C2, IsInvalid) (Syntax: 'New C2() Fr ... lo World!"}')
-          Arguments(0)
-          Initializer: 
-            IObjectOrCollectionInitializerExpression (OperationKind.ObjectOrCollectionInitializerExpression, Type: C2, IsInvalid) (Syntax: 'From {"Hello World!"}')
-              Initializers(1):
-                  ICollectionElementInitializerExpression (AddMethod: Sub C2.Add(p As System.String)) (IsDynamic: False) (OperationKind.CollectionElementInitializerExpression, Type: System.Void, IsInvalid, IsImplicit) (Syntax: '"Hello World!"')
-                    Arguments(1):
-                        ILiteralExpression (OperationKind.LiteralExpression, Type: System.String, Constant: "Hello World!", IsInvalid) (Syntax: '"Hello World!"')
+        ILocalInitializer (OperationKind.LocalInitializer, IsInvalid) (Syntax: '= New C2()  ... lo World!"}')
```

Comment:
>= New C2()  ... lo World!"} [](start = 80, length = 27)

It looks like the syntax node is correct, but why the  string is truncated before the With and doesn't include any indication of truncation?

Reply:
Filed [https://github.com/dotnet/roslyn/issues/22575](https://github.com/dotnet/roslyn/issues/22575)

---
In reply to: [143043591](https://github.com/dotnet/roslyn/pull/22546#discussion_r143043591) [](ancestors = 143043591)